src/main/resources/JUnitStandaloneTest.ftl

```diff
@@ -0,0 +1,17 @@
+package ${package};
+
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import static org.junit.Assert.*;
+import static org.hamcrest.core.Is.*;
```

Comment:
I think we should also include class under test ` import ${packageImport}.${ClassToTest};`